site/docs/supported-endpoints.md

```diff
@@ -0,0 +1,145 @@
+---
+id: supported-endpoints
+title: Supported OpenAI API Endpoints
+sidebar_position: 9
+---
+
+The Envoy AI Gateway provides OpenAI-compatible API endpoints for routing and managing LLM/AI traffic. This page documents which OpenAI API endpoints are currently supported and their capabilities.
+
+## Overview
+
+The Envoy AI Gateway acts as a proxy that accepts OpenAI-compatible requests and routes them to various AI providers. While it maintains compatibility with the OpenAI API specification, it currently supports a subset of the full OpenAI API.
+
+## Supported Endpoints
+
+### Chat Completions
+
+**Endpoint:** `POST /v1/chat/completions`
+
+**Status:** ✅ Fully Supported
+
+**Description:** Create a chat completion response for the given conversation.
+
+**Features:**
+- ✅ Streaming and non-streaming responses
+- ✅ Function calling
+- ✅ Response format specification (including JSON schema)
+- ✅ Temperature, top_p, and other sampling parameters
+- ✅ System and user messages
+- ✅ Model selection via request body or `x-ai-eg-model` header
+- ✅ Token usage tracking and cost calculation
+- ✅ Provider fallback and load balancing
+
+**Supported Providers:**
+- OpenAI
+- AWS Bedrock (with automatic translation)
+- Azure OpenAI (with automatic translation)
+- Any OpenAI-compatible provider (Groq, Together AI, Mistral, etc.)
+
+**Example:**
+```bash
+curl -H "Content-Type: application/json" \
+  -d '{
+    "model": "gpt-4o-mini",
+    "messages": [
+      {
+        "role": "user",
+        "content": "Hello, how are you?"
+      }
+    ]
+  }' \
+  $GATEWAY_URL/v1/chat/completions
+```
+
+### Embeddings
+
+**Endpoint:** `POST /v1/embeddings`
+
+**Status:** ✅ Fully Supported
+
+**Description:** Create embeddings for the given input text.
+
+**Features:**
+- ✅ Single and batch text embedding
+- ✅ Model selection via request body or `x-ai-eg-model` header
+- ✅ Token usage tracking and cost calculation
+- ✅ Provider fallback and load balancing
+
+**Supported Providers:**
+- OpenAI
+- Any OpenAI-compatible provider that supports embeddings
+
+## Provider-Endpoint Compatibility
+
+The following table shows which providers support which endpoints:
```

Comment:
did you really check each provider actually supports the embedding? E.g. https://api-docs.deepseek.com/ deepseek seems not support embedding. could you double check each website and see if it works?